src/components/AvatarCropModal/ImageCropView.tsx

```diff
@@ -1,60 +1,66 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
 import {PanGestureHandler} from 'react-native-gesture-handler';
+import type {GestureEvent, PanGestureHandlerEventPayload} from 'react-native-gesture-handler';
 import Animated, {interpolate, useAnimatedStyle} from 'react-native-reanimated';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import ControlSelection from '@libs/ControlSelection';
-import gestureHandlerPropTypes from './gestureHandlerPropTypes';
+import type {SelectionElement} from '@libs/ControlSelection/types';
+import type IconAsset from '@src/types/utils/IconAsset';
 
-const propTypes = {
+type ImageCropViewProps = {
     /** Link to image for cropping   */
-    imageUri: PropTypes.string,
+    imageUri: string;
 
     /** Size of the image container that will be rendered */
-    containerSize: PropTypes.number,
+    containerSize: number;
 
     /** The height of the selected image */
-    originalImageHeight: PropTypes.shape({value: PropTypes.number}).isRequired,
+    originalImageHeight: {
+        value: number;
+    };
 
     /** The width of the selected image */
-    originalImageWidth: PropTypes.shape({value: PropTypes.number}).isRequired,
+    originalImageWidth: {
+        value: number;
+    };
 
     /** The rotation value of the selected image */
-    rotation: PropTypes.shape({value: PropTypes.number}).isRequired,
+    rotation: {
+        value: number;
+    };
 
     /** The relative image shift along X-axis */
-    translateX: PropTypes.shape({value: PropTypes.number}).isRequired,
+    translateX: {
+        value: number;
+    };
 
     /** The relative image shift along Y-axis */
-    translateY: PropTypes.shape({value: PropTypes.number}).isRequired,
+    translateY: {
+        value: number;
+    };
 
     /** The scale factor of the image */
-    scale: PropTypes.shape({value: PropTypes.number}).isRequired,
+    scale: {
+        value: number;
+    };
 
     /** React-native-reanimated lib handler which executes when the user is panning image */
-    panGestureEventHandler: gestureHandlerPropTypes,
+    panGestureEventHandler: (event: GestureEvent<PanGestureHandlerEventPayload>) => void;
```

Comment:
```suggestion
    panGestureEventHandler?: (event: GestureEvent<PanGestureHandlerEventPayload>) => void;
```

According to propTypes.